src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -306,6 +306,12 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
    */
   private _setValueAndClose(event: MdOptionSelectionChange | null): void {
     if (event) {
+      this.autocomplete.options.forEach((option) => {
+        if (option != event.source) {
+          option.deselect();
```

Comment:
Calling `deselect()` here will emit a selection change event for every option on the list. That's fine, but it will change the behavior of subscriptions to `optionSelections`. Instead of getting 1 event on selection, people will get 1 event per option (so, 50 events for a 50 option list). So I'd also add a filter to `optionSelections` to ensure we're only catching events from options that have selected set to true.

Also can you move this out into its own function?